Code/Tools/TestImpactFramework/Frontend/Console/Common/Code/Include/Static/TestImpactCommandLineOptionsUtils.h

```diff
@@ -30,26 +30,21 @@ namespace TestImpact
     //! Attempts to parse the specified binary state option.
     template<typename T>
     AZStd::optional<T> ParseBinaryStateOption(
-        const AZStd::string& optionName,
-        const AZStd::pair<OptionValue<T>,
-        OptionValue<T>>& state, const AZ::CommandLine& cmd)
+        const AZStd::string& optionName, const AZStd::pair<OptionValue<T>, OptionValue<T>>& state, const AZ::CommandLine& cmd)
     {
-        if (const auto numSwitchValues = cmd.GetNumSwitchValues(optionName);
-            numSwitchValues)
+        if (const auto numSwitchValues = cmd.GetNumSwitchValues(optionName); numSwitchValues)
```

Comment:
suggestion: Rather than relying on an implicit `0` check here, having this be `numSwitchValues == 0` would be clearer (repeated in a few other spots)